src/human.ts

```diff
@@ -0,0 +1,170 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** @module common-sk/modules/human
+ *  @description Utitities for working with human friendly I/O.
+ */
+
+interface Delta {
+  units: string;
+  delta: number;
+}
+
+const TIME_DELTAS: Delta[] = [
+  { units: 'w', delta: 7 * 24 * 60 * 60 },
+  { units: 'd', delta: 24 * 60 * 60 },
+  { units: 'h', delta: 60 * 60 },
+  { units: 'm', delta: 60 },
+  { units: 's', delta: 1 }
+];
+
+/** @constant {number} */
```

Comment:
Both books were a little iffy using enums. Also the functions are usable w/o being that strict, for example if the input is in MB vs MiB you can just call bytes(12, 1000*1000) and will work correctly. If we are too strict then the above code wouldn't work and would require going back and adding more values in the enum.

Reply:
How about making the functions take a `StorageUnit | number` parameter? This would offer the best of both worlds: a nice, intellisense-friendly list of curated, self documenting values; with the option to specify a custom value if needed.

Reply:
Will consider for a later PR.